src/views/LimitOrders/hooks/useGelatoLimitOrdersHistory.ts

```diff
@@ -52,13 +55,17 @@ const useOpenOrders = (turnOn: boolean): Order[] => {
     startFetch ? ['gelato', 'openOrders'] : null,
     async () => {
       try {
-        const orders = await gelatoLimitOrders.getOpenOrders(account.toLowerCase(), false)
+        const [openOrders, canOrders, exeOrders] = await Promise.all([
```

Comment:
We should fetch all the orders to determine order status updated to cancelled or executed. (Cancelled added as wel to possibility to cancel order from another device)